openedx/features/qverse_features/registration/validators.py

```diff
@@ -0,0 +1,32 @@
+"""
+Validation utils for qverse registration application.
+"""
+from django.core.exceptions import ValidationError
+
+
+def validate_file_extension(value):
+    """
+    Validates that file is a valid csv file by checking its extension.
+    """
+    if not value.name.endswith('.csv'):
+        raise ValidationError(u'Please upload a valid csv file.')
+
+
+def validate_current_level(value):
+    """
+    Validates the provided current level.
+    """
+    # to avoid circular import
+    from .models import MAX_LEVEL_CHOICES
+    if not 1 <= value <= MAX_LEVEL_CHOICES:
+        raise ValidationError(u'Please provide valid current level.')
```

Comment:
where this message would be shown? If in logs, can we make this and in the next method more descriptive. e.g `Given current level: <id> is not valid.` We may need to handle such scenarios in the signal.

These validators won't affect the registration form, because we would have dropdown there.

Reply:
No, but if somebody is doing submissions through postman he can truly make our database dirty so to avoid that scenario I have added these validators.

And theses message would be shown on GUI.

Reply:
Agree on the first part, but didn't get when these messages will be shown on GUI?

Reply:
discussed. And changed the `raise ValidationErro()` statements.